plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/AccountsPanel.java

```diff
@@ -29,25 +30,24 @@
 import org.eclipse.swt.events.SelectionAdapter;
 import org.eclipse.swt.events.SelectionEvent;
 import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.layout.GridData;
 import org.eclipse.swt.widgets.Button;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.Control;
 import org.eclipse.swt.widgets.Label;
 import org.eclipse.swt.widgets.Shell;
 
-import java.util.ArrayList;
-import java.util.List;
-
 /**
  * A panel listing all currently logged-in accounts. The panel allows adding new accounts and
  * logging out all accounts.
  */
 public class AccountsPanel extends PopupDialog {
 
-  private IGoogleLoginService loginService;
+  private final IGoogleLoginService loginService;
 
   @VisibleForTesting Button logOutButton;
-  @VisibleForTesting List<Label> accountLabels = new ArrayList<>();
+  @VisibleForTesting final List<Label> nameLabels = new ArrayList<>();
```

Comment:
If you want to test that items are present, set an id on the labels (to identify their label type), and use SWTBot to pull out labels.  You can either use the SWTBot default ID key (`org.eclipse.swtbot.widget.key`) or use the E4 CSS class key (`org.eclipse.e4.ui.css.CssClassName`).  Then you should be able to use SWTBot's `labelWithId(key,value,index)` to pull out values and check order.